packages/preview/subpar/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+entrypoint = "src/lib.typ"
+name = "subpar"
+version = "0.1.0"
+compiler = "0.11.0"
+authors = ["tinger <me@tinger.dev>"]
+repository = "https://github.com/tingerrr/subpar"
+description = "Create sub figures easily."
+categories = ["components", "model"]
+keywords = ["figures", "sub"]
+license = "MIT"
+exclude = ["examples", "doc", "README.md"]
```

Comment:
Your README should not be excluded I think.

Reply:
Wasn't much of an issue before, as people usually check either the universe page or the GitHub repo out, but I'll include it.